lightkurve/lightcurve.py

```diff
@@ -673,8 +677,12 @@ def normalize(self, unit='unscaled'):
 
         # Create a new light curve instance and normalize its values
         lc = self.copy()
-        lc.flux = lc.flux / median_flux
-        lc.flux_err = lc.flux_err / median_flux
+        if method == 'median':
+            denominator = median_flux
+        elif method == 'mean':
+            denominator = np.nanmean(self.flux)
+        lc.flux = lc.flux / denominator
```

Comment:
Please replace with 
`lc /= denominator` 
We have error propagation in the divide, and I'd rather we didn't have two places in the code where we could introduce bugs